MarkBind/markbind
# Add copy buttons to code blocks

**What is the purpose of this pull request? (put "X" next to an item, remove the rest)**

• [X] Enhancement to an existing feature

Fixes #562.

**What is the rationale for this request?**

Adding copy buttons to code blocks will allow users to copy code snippets with a single click. 

**What changes did you make? (Give an overview)**

1. Added copy buttons to all code blocks (`Page.js`, `markbind.css`). 
2. Handled copy to clipboard on click (`setup.js`) using the minified version of [clipboard.js](https://github.com/zenorocha/clipboard.js) as an imported asset.

![screenshot_2019-01-24 markbind - user guide - quick start](https://user-images.githubusercontent.com/22164211/51674772-97346300-200c-11e9-883f-69eb52d77679.png)

**Testing instructions:**

Build the docs website and check the copy buttons for code blocks are working correctly.
